doc/ota_updates/readme.rst

```diff
@@ -102,10 +102,10 @@ If you receive either of the following messages in the IDE window, the signing w
     ... or ...
     Not signing the generated binary
 
-Manual Signing Binaries
-^^^^^^^^^^^^^^^^^^^^^^^
+Manual Signing of Binaries
+^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
One ^ short, I think the automated docmaker will complain.  It's a pain!

Reply:
Thanks, didn't know it was that picky.